.github/workflows/golangci-lint.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
         uses: golangci/golangci-lint-action@v2
         with:
           # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
-          version: v1.41
+          version: v1.45.2
```

Comment:
we can just rebase this PR after merging the linter fix

Reply:
rebased